src/client.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  Client as ClientW3f,
+  Keystore,
+  ZeroBN,
+} from '@w3f/polkadot-api-client';
+import { StakingLedger } from '@polkadot/types/interfaces'
```

Comment:
@fgimenez is there a way to use the imported imports from polkadot-api-client for the sake of avoiding versions mismatch ? 
Is it even a good idea ?

Reply:
i think it would be ok in order to align dependencies and unify imports, but polkadot-api-client would need to export them explicitly.